pkg/operator/controller/instance_controller.go

```diff
@@ -47,7 +47,7 @@ const (
 	DataPlaneName     = "cl-dataplane"
 	GoDataPlaneName   = "cl-go-dataplane"
 	IngressName       = "cl-external-svc"
-	OperatorNameSpace = "clusterlink-operator" // TODO -should be removed
+	OperatorNamespace = "clusterlink-operator"
```

Comment:
Why is the default operator namespace different than the instance namespace?

Reply:
Usually, the operator runs in a different namespace with more privileges (like in openshift there is an openshift-operator namespace)